client/src/components/client-page/index.jsx

```diff
@@ -0,0 +1,232 @@
+// import * as React from "react";
```

Comment:
Make all buttons on this and the edit page clickable (even if the action doesn't actually do anything)